### PR TITLE
Change conversation text request to order

### PR DESCRIPTION
**Pull request by thomasmalbaux**
After discussions, we understood that the term "Request" isn't the best one to define a transaction/order after a payment has been done. This lead to some confusion among users and was reported a few times.

Instead of using sentences like "The request has been completed", it now says "The order has been completed".

I did my best to update all related sentences (layout, inbox, email notifications...).

### CODE REVIEW

**Comment by thomasmalbaux** [comment]
Also I didn't check if some tests are using the "request" term so good to wait and see if they pass.

File: config/locales/en.yml [hunk 1]
```diff
@@ -625,9 +625,9 @@ en:
       accepted_offer: "Offer accepted."
       rejected_request: "Request rejected."
```
Comment (rap1ds) [inline config/locales/en.yml]: What about this?

File: config/locales/en.yml [hunk 2]
```diff
@@ -638,9 +638,9 @@ en:
       accepted_offer: "accepted the offer"
       rejected_request: "rejected the request"
```
Comment (rap1ds) [inline config/locales/en.yml]: What about this?

File: config/locales/en.yml [hunk 3]
```diff
@@ -830,18 +830,18 @@ en:
       account_ready: "You are ready to receive payments"
       message: "Your payment information has been confirmed and you are now ready to receive payments in %{service_name}."
     confirm_reminder:
-      you_have_not_yet_confirmed_or_canceled_request: "You have not yet confirmed or canceled the request %{request_link}. If the request has been completed, you should confirm that. After that you can give feedback to %{other_party_given_name}."
+      you_have_not_yet_confirmed_or_canceled_request: "You have not yet confirmed or canceled the order %{request_link}. If the order has been completed, you should confirm that. After that you can give feedback to %{other_party_given_name}."
       you_have_not_yet_confirmed_or_canceled_request_escrow_automatic_confirmation: "You have not yet confirmed or canceled the request %{request_link}. We will transfer the money to %{other_party_given_name} only when a) you have marked the request completed or b) %{days_to_automatic_confirmation} days have passed since you paid. After marking the request as completed you can give feedback to %{other_party_given_name}."
```
Comment (rap1ds) [inline config/locales/en.yml]: Request

File: config/locales/en.yml [hunk 4]
```diff
@@ -830,18 +830,18 @@ en:
...
       remember_to_confirm_request: "Remember to confirm or cancel a request"
-      if_will_not_happen_you_should_cancel: "If you think this request will not be completed for one reason or another, you can %{cancel_it_link}."
+      if_will_not_happen_you_should_cancel: "If you think this order will not be completed for one reason or another, you can %{cancel_it_link}."
       if_will_not_happen_you_should_cancel_escrow: "If you think this request will not be completed for one reason or another, you can %{cancel_it_link}. This will notify the admin who will then contact you for a refund."
```
Comment (rap1ds) [inline config/locales/en.yml]: Request

File: config/locales/en.yml [hunk 5]
```diff
@@ -1361,10 +1361,9 @@ en:
       reply_cannot_be_empty: "You can't send an empty message"
       reply_sent: "Reply sent successfully"
       request_accepted: "Request accepted"
```
Comment (rap1ds) [inline config/locales/en.yml]: Request

File: config/locales/en.yml [hunk 6]
```diff
@@ -902,7 +902,7 @@ en:
       your_request_was_rejected: "Your request was rejected"
```
Comment (rap1ds) [inline config/locales/en.yml]: Request

File: config/locales/en.yml [hunk 7]
```diff
@@ -1361,10 +1361,9 @@ en:
       reply_cannot_be_empty: "You can't send an empty message"
       reply_sent: "Reply sent successfully"
       request_accepted: "Request accepted"
-      request_confirmed: "Request completed"
-      request_closed: "Request closed"
+      request_confirmed: "Order completed"
       request_rejected: "Request rejected"
```
Comment (rap1ds) [inline config/locales/en.yml]: Request

File: config/locales/en.yml [hunk 8]
```diff
@@ -601,8 +601,8 @@ en:
       paypal_fee_info: "* Excludes <a id='%{fee_info_link_id}' href='#'>PayPal's payment processing fee</a>"
       shipping_price_label: "Shipping:"
     confirm:
-      confirm_description: "If your request has been fulfilled you should confirm it as done. Then you can give feedback to the other party."
-      cancel_description: "If your request was not fulfilled, you can mark it as 'disputed'. You can still give feedback to the other party and describe what happened."
+      confirm_description: "If your order has been fulfilled you should confirm it as done. Then you can give feedback to the other party."
+      cancel_description: "If your order was not fulfilled, you can mark it as 'disputed'. You can still give feedback to the other party and describe what happened."
       cancel_payed_description: "If your request was accepted but you're having second thoughts, you can still cancel the request before the payment."
```
Comment (rap1ds) [inline config/locales/en.yml]: Request

File: config/locales/en.yml [hunk 9]
```diff
@@ -731,8 +731,8 @@ en:
       waiting_for_listing_author_to_accept_offer: "Waiting for %{listing_author_name} to accept the offer"
       waiting_for_listing_author_to_accept_request: "Waiting for %{listing_author_name} to accept the request. As soon as %{listing_author_name} accepts, you will be charged."
```
Comment (rap1ds) [inline config/locales/en.yml]: Request

File: config/locales/en.yml [hunk 10]
```diff
@@ -731,8 +731,8 @@ en:
...
       waiting_for_you_to_accept_request: "Waiting for you to accept the request"
```
Comment (rap1ds) [inline config/locales/en.yml]: Request

**Comment by thomasmalbaux** [comment]
Reviewed Mikko suggestions and updated one more sentence.
Some "request" or "offer" terms remain as the idea right now is (only) to rename them "order" only for steps after the actual payment.

**Comment by rap1ds** [comment]
Looks good to me

**Pull request merged**